wiki/Game_modifier/Easy/fr.md

```diff
@@ -70,7 +70,7 @@ Dans le mode [osu!mania](/wiki/Game_mode/osu!mania), les effets sont les mêmes
 ## Le saviez-vous ?
 
 - Lorsque vous regardez un replay où une vie supplémentaire est utilisée, l'animation de recharge de la santé sera ignorée et le replay continuera comme si le joueur jouait avec le mod [No Fail](/wiki/Game_modifier/No_Fail) activé.
-- En mod [Multi](/wiki/Gameplay/Multiplayer), l'effet des "vies" ne fonctionnera pas en mod [Tag Co-op ou Tag Team Vs.](/wiki/Gameplay/Multiplayer#tag-coop-tag-team-vs)
+- En mod [Multi](/wiki/Gameplay/Multiplayer), l'effet des "vies" ne fonctionnera pas en mod [Tag Co-op ou Tag Team Vs.](/wiki/Gameplay/Multiplayer#tag-coop-/-tag-team-vs)
```

Comment:
i was wondering why this wasn't caught by the link checker, but then i actually checked the french article and realised it was outdated, so this is correct <.<